src/current/v23.2/migration-overview.md

```diff
@@ -98,7 +98,7 @@ As part of [migration preparations](#prepare-for-migration), you will have alrea
 
 This method adds a fallback plan to the simple [all-at-once](#all-at-once-no-rollback) cutover.
 
-In addition to moving data to CockroachDB, data is also replicated from CockroachDB back to the source database in case you need to roll back the migration. Continuous replication is already possible when performing a [zero-downtime migration](#zero-downtime) that dual writes to both databases. Otherwise, you will need to ensure that data is replicated in the reverse direction at cutover. The challenge is to find a point at which both the source database and CockroachDB are in sync, so that you can roll back to that point. You should also avoid falling into a circular state where updates continuously travel back and forth between the source database and CockroachDB.
+In addition to moving data to CockroachDB, data is also replicated from CockroachDB back to the source database in case you need to roll back the migration. Continuous replication is already possible when performing a [live migration](#live-migration) that dual writes to both databases. Otherwise, you will need to ensure that data is replicated in the reverse direction at cutover. The challenge is to find a point at which both the source database and CockroachDB are in sync, so that you can roll back to that point. You should also avoid falling into a circular state where updates continuously travel back and forth between the source database and CockroachDB.
```

Comment:
Since we're touching this I also wonder if we should state that final recommendation more strongly and in bold, could be missed when skimming:

 **Avoid falling into a circular state where ... **